files/scripts/swss.sh

```diff
@@ -149,15 +151,31 @@ start() {
         clean_up_tables STATE_DB "'PORT_TABLE*', 'MGMT_PORT_TABLE*', 'VLAN_TABLE*', 'VLAN_MEMBER_TABLE*', 'LAG_TABLE*', 'LAG_MEMBER_TABLE*', 'INTERFACE_TABLE*', 'MIRROR_SESSION*', 'VRF_TABLE*', 'FDB_TABLE*', 'FG_ROUTE_TABLE*', 'BUFFER_POOL*', 'BUFFER_PROFILE*', 'MUX_CABLE_TABLE*'"
     fi
 
+    # Check asic status before starting docker
```

Comment:
Thanks for the updates. 

Do we sill need to check_asic_status() in both wait and start, as the following is the sequence in which systemd tries to start a service

```
ExecStartPre=/usr/local/bin/swss.sh start
ExecStart=/usr/local/bin/swss.sh wait
ExecStop=/usr/local/bin/swss.sh stop
```
So doing it either in end of start(), OR beginning of wait() should do ?

Note: This is applicable to all services updated in this PR viz swss, syncd, teamd, lldp.

Reply:
For fixed platforms, I did not want to change the timing of the services since the asic_status.py does not run and systemd has control of it under ExecStartPre (and not under ExecStart).

Reply:
I would still prefer to keep it only in one place in the start() alone for two reasons 

1. In check_asic_status() we anyways exit if it is fixed platform , due to this check [ $chassis_sup -eq 0 ]  **( we can remove the additional parameter start/wait being passed )**. So there is no additional delay introduced for single asic platforms.
2. If we do it in start before calling /usr/bin/${SERVICE}.sh start $DEV, it is a good place as we don't even create the docker and dependent services
  
Let me know what you think.

Reply:
It does not work if we run the asic_status.py() in the start() and we keep it waiting for asics not present in the system. Systemd will not allow snmp to start unless all asics complete ExecStartPre. That was the reason we decided to run it in start() for fixed platforms and chassis line-cards. And, run it in wait() for chassis supervisor.

Reply:
Fixed. Please review.